client/src/main/java/com/paypal/selion/reports/runtime/Gatherer.java

```diff
@@ -41,7 +44,7 @@ static String saveGetLocation(WebDriver driver) {
                 location = driver.getCurrentUrl();
             }
         } catch (Exception exception) {
-            logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
+            logger.log(Level.FINEST, "Current location couldn't be retrieved by getCurrentUrl().", exception);
```

Comment:
Okay, that makes sense.  I still fee we can do better :)

Though, I'm generally not in favor of `instanceof`, would doing an `instanceof` check on `driver` help in any way here? Sure, we won't be able to rely on `driver` with absolute certainty to tell us if it is a mobile `driver` AND we won't know anything about the `app` under test. Just a thought. Otherwise, I my only suggestion for improvement is to change the logged message a bit to add something like `this can be safely ignored if testing a non-web mobile application`